applications/asset_tracker/src/main.c

```diff
@@ -237,6 +237,11 @@ static void set_gps_enable(const bool enable);
 static bool data_send_enabled(void);
 static void connection_evt_handler(const struct cloud_event *const evt);
 static void no_sim_go_offline(struct k_work *work);
+#if defined(CONFIG_AGPS)
+/* Converts the A-GPS data request from GPS driver to GNSS API format. */
+static void agps_request_convert(struct nrf_modem_gnss_agps_data_frame *dest,
+	const struct gps_agps_request *src);
+#endif
```

Comment:
Because `agps_request_convert()` is now used also in `manage_pgps()`, `agps_request_convert()` is also needed when `CONFIG_NRF_CLOUD_PGPS` is enabled but `CONFIG_AGPS` disabled (P-GPS only). The same change is also needed for the function definition.
```suggestion
#if defined(CONFIG_AGPS) || defined(CONFIG_NRF_CLOUD_PGPS)
/* Converts the A-GPS data request from GPS driver to GNSS API format. */
static void agps_request_convert(struct nrf_modem_gnss_agps_data_frame *dest,
	const struct gps_agps_request *src);
#endif
```

Reply:
Nice, thanks.
Should i go ahead and make these changes to ATv1?  
Or just remove ATv1 changes from this PR since it is deprecated.

Reply:
@jhn-nordic just made this PR against ATv1, so I think you should keep these fixes as well.
https://github.com/nrfconnect/sdk-nrf/pull/6150